### ISSUE TITLE
Can't find a way to render json

### ISSUE THREAD

**Issue by ozankasikci** [marmelab/admin-on-rest#345]
Hi,

None of the components seem to be supporting a source in json format. I also need to be able to edit json fields of the resource. Am i missing something or there is no way to render json?

Thanks.

**Comment by fzaninotto** [comment]
Admin-on-rest doesn't provide one, but that doesn't mean that it's hard. In fact, to display the raw JSON data, it's a one-liner:

```js
const RawJsonField = ({ record, source }) => <span>{JSON.stringify(record[source])}</span>;
```

You can also access nested fields in all core Field components:

```js
<TextField source="foo.bar.baz" />
```

If you're looking for a JSON Input, there is none either. But you can write one easily using a third-party library like [react-codemirror](https://github.com/JedWatson/react-codemirror).

The documentation clearly explains how to build a [Custom Field](https://marmelab.com/admin-on-rest/Fields.html#writing-your-own-field-component) and a [Custom Input](https://marmelab.com/admin-on-rest/Inputs.html#writing-your-own-input-component).

To summarize, the answer to your question "can't find a way" is: "Look harder: you can do it yourself".
